SofascoreAcademy/SofascoreAcademy/Controller/ViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 07.03.2024..
+//
+
+import SnapKit
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .lightGray
+        
+        let league = createMockLeague()
+        let matches = createMockMatches()
+        
+        let leagueComponent: LeagueComponentView = LeagueComponentView(league: league, matches: matches)
+        
+        view.addSubview(leagueComponent)
+        
+        leagueComponent.snp.makeConstraints() {
+            $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).inset(30)
+            $0.left.equalTo(view.safeAreaLayoutGuide.snp.left)
```

Comment:
Za lijevo ti ne trebaju safeAreaLayoutGuide. 

I možeš koristiti leading/trailing umjesto left/right, to je standard... možda ćeš jednog dana trebati arabsku lokalizaciju pa će ti dobro doći :)

```suggestion
            $0.leading.equalToSuperview()
```